src/map.cpp

```diff
@@ -238,95 +237,96 @@ bool Map::placeCreature(const Position& centerPos, Creature* creature, bool exte
 
 	int32_t index = 0;
 	uint32_t flags = 0;
-	Item* toItem = nullptr;
+	std::shared_ptr<Item> toItem = nullptr;
```

Comment:
Could this be `auto`
```suggestion
	auto toItem = std::shared_ptr<Item>(nullptr);
```